doc/functions/block.rst

```diff
@@ -11,6 +11,14 @@ times, use the ``block`` function:
     <h1>{{ block('title') }}</h1>
 
     {% block body %}{% endblock %}
+    
+The block can also be used without an endblock when give the content of the block directly by string or variable:
+
+.. code-block:: jinja
+
+    {% block title 'title' %}
```

Comment:
This document is about `block()` as a function. This new documentation likely fits better with `{% block %}` tags section https://github.com/twigphp/Twig/blob/1.x/doc/tags/block.rst